doc/users/cmake_dealii.html

```diff
@@ -894,6 +895,44 @@ <h3> Selecting a compiler </h3>
     </p>
 
 
+    <a name="compileccache"></a>
+    <h3> Compiling using CCache </h3>
+    <p>
+      It is possible for the library to be built using the
+      <a href="https://ccache.dev/">CCache</a> ("compiler cache") tool.
```

Comment:
I think it would be better to start by describing what ccache is and then quickly mention that we support using it so that users get the gist of it by reading the first sentence.

Reply:
Good suggestion, thanks. I'll swap these two paragraphs around.